src/components/FiveCardLayout.js

```diff
@@ -0,0 +1,39 @@
+import React, {Component, PropTypes} from 'react'
+import Card from './Card'
+import CardBack from './CardBack'
+import _ from 'lodash'
+
+export default class FiveCardLayout extends Component {
+
+  static propTypes = {
+    cards: PropTypes.array.isRequired,
+
+  }
+
+  renderCards() {
+    // assign positions
+    // render Cards or CardBacks
+
+    return this.props.cards.map( card => {
+      return Object.assign( {}, card )
+    }).map( (card, index) => {
+      if( card.faceUp ) {
```

Comment:
a ternary might be easier here: 
```js
return card.faceUp
  ? <Card 
    {...card} 
    positionInfo={this.props.positionInfos[index]}
    flippedIndex={this.props.flippedIndex}
    positionTitle={this.props.positionTitles[index]}
    key={`card-${index}`} 
  />
  : <CardBack {...card} key={`card-${index}`} />
```

but why do you need to render a `Card` or a `CardBack`. Doesnt the `Card` component take a `flippedIndex` property? It seems like `Card` should know if its flipped and decided internally to show the front or back image of the card